src/Sulu/Bundle/AutomationBundle/Resources/public/js/services/task-manager.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['underscore', 'services/husky/util'], function(_, Util) {
+
+    'use strict';
+
+    var url = _.template('/admin/api/tasks<% if(!!id) { %>/<%= id %><% } %>');
+
+    /** @constructor **/
+    function Manager() {
+    }
+
+    Manager.prototype = {
+        getUrl: function(entityClass, entityId) {
+            return url({id: null}) + '?entityClass=' + entityClass + '&entity-id=' + entityId;
+        },
+
+        load: function(id) {
+            return Util.load(url({id: id}));
+        },
+
+        save: function(data) {
+            return Util.save(url({id: data.id || null}), !!data.id ? 'PUT' : 'POST', data);
+        },
+
+        deleteItems: function(ids) {
+            return Util.save(url({id: null}) + '?ids=' + ids.join(','), 'DELETE');
+        }
+    };
+
+    return new Manager();
```

Comment:
Any reason this "complicated" structure is required? Why not simply returning an object?